tests/plugin_tests/manual.test_load_configuration_defaults_plugin/main.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
I'm wondering if `test/plugin_tests` is a good place to put these. Maybe they should live under `app/plugin_tests` given that they are testing the plugin API implementation.

@reflog @lieut-data What do you think?

Reply:
I don't want them to be in the main tree mostly because they are not 'real' go programs. The are preprocessed and parts of them are replaced during the test running

Reply:
If we want to make this clear, we could add a `// +build ignore` flag.

Reply:
I'm cool with that idea. @lieut-data - thoughts?

Reply:
Actually, I don't know if the build flag will work, as we _are_ building these files.

Reply:
How hard would it be to skip the pre-processing, and pass that data some other way? (i.e. via plugin config instead)